core/src/main/scala/org/apache/spark/util/collection/BitSet.scala

```diff
@@ -32,6 +34,10 @@ class BitSet(numBits: Int) extends Serializable {
    */
   def capacity: Int = numWords * 64
 
+  def clear(): Unit = {
+    util.Arrays.fill(words, 0L)
```

Comment:
Hm, sorry, I take that back. There is no equivalent in Scala that I can find, that sets an existing array's values. Can I suggest thought that importing just `java.util` might look confusing, so either just import it (there is no  Scala `Arrays` to mix it up with) or if this is just one usage, write `java.util.Arrays.fill(...)`